### PR TITLE
More details on how to set up coc

### PR DESCRIPTION
**Pull request by mark-i-m**
I spent ~1 hour trying to figure this out. It's all pretty simple stuff, but very annoying...

### CODE REVIEW

File: docs/user/README.md [hunk 1]
```diff
@@ -92,7 +92,13 @@ to load path and require it in `init.el`
 ## Vim and NeoVim
 
 * Install coc.nvim by following the instructions at [coc.nvim]
+  - You will need nodejs installed.
```
Comment (matklad) [inline docs/user/README.md]: Why would one need nodejs in this case?

Comment (matklad) [reply docs/user/README.md]: Ah, so coc itself is implemented in nodejs. I have no idea how that actually works.

**Comment by matklad** [comment]
bors r+

Thanks!

File: docs/user/README.md (hunk 1 above)
Comment (mark-i-m) [reply docs/user/README.md]: Yes, I believe the client-side of the LSP protocol in coc is implemented in js or something. Either way, their installation instruction require it: https://github.com/neoclide/coc.nvim/wiki/Install-coc.nvim

**Pull request merged**